tests/testthat/test_featsel_filters.R

```diff
@@ -73,6 +73,29 @@ test_that("randomForestSRC_var.select filter handles user choices correctly", {
   )
 })
 
+test_that("Custom threshold function for filtering works correctly", {
+	biggest_gap = function(values, diff) {
+	  gap_size = 0
+	  gap_location = 0
+
+	  for (i in (diff+1):length(values)) {
+			gap = values[[i - diff]] - values[[i]]
+			if (gap > gap_size) {
+				gap_size = gap
+				gap_location = i - 1
+			}
+	  }
+	  return(gap_location)
+	}
+
+  ftask = filterFeatures(task = multiclass.task,
+                         method = "variance",
+                         fun = biggest_gap,
+                         fun.args = list("diff" = 1)
```

Comment:
Thanks.

Please style the changed files according to our [mlr-style](https://github.com/mlr-org/mlr3/wiki/style-guide#styler-mlr-style).

Reply:
OK done.